ledger-service/http-json/src/main/scala/com/digitalasset/http/domain.scala

```diff
@@ -153,15 +153,13 @@ package domain {
   )
 
   final case class GetActiveContractsRequest(
-      // TODO #14727 remove .Template for subscriptions
-      templateIds: OneAnd[Set, ContractTypeId.Template.OptionalPkg],
+      templateIds: OneAnd[Set, ContractTypeId.OptionalPkg],
       query: Map[String, JsValue],
       readAs: Option[NonEmptyList[Party]],
   )
 
   final case class SearchForeverQuery(
-      // TODO #14727 remove .Template for subscriptions
-      templateIds: OneAnd[Set, ContractTypeId.Template.OptionalPkg],
+      templateIds: OneAnd[Set, ContractTypeId.OptionalPkg],
```

Comment:
Can we leave this as-is and do it in #14817 instead?

Reply:
Sure.